examples/prometheus/README.md

```diff
@@ -119,26 +119,22 @@ Returns PLEG (pod lifecycle event generator) latency metrics.  This represents t
 
 ### OpenShift build related queries
 
-> count(openshift_build_running_phase_start_time_seconds{} < time() - 600)
+> count(openshift_build_active_time_seconds{phase="running"} < time() - 600)
 
 Returns the number of builds that have been running for more than 10 minutes (600 seconds).
 
-> count(openshift_build_new_pending_phase_creation_time_seconds{} < time() - 600)
+> count(openshift_build_active_time_seconds{phase="pending"} < time() - 600)
 
 Returns the number of build that have been waiting at least 10 minutes (600 seconds) to start.
 
-> sum(openshift_build_failed_phase_total{})
+> sum(openshift_build_phase_reason_total{phase="failed"})
 
 Returns the number of failed builds, regardless of the failure reason.
 
-> sum(openshift_build_terminal_phase_total{phase="complete"})
+> sum(openshift_build_phase_reason_total{phase="complete"})
 
 Returns the number of successfully completed builds.
 
-> openshift_build_failed_phase_total{}
-
-Returns the latest totals, per failure reason, for any failed builds.
-
-> openshift_build_failed_phase_total{} offset 5m
+> openshift_build_phase_reason_total{phase="failed"} offset 5m
```

Comment:
add an example of querying on "count builds that failed due to reason X"

Reply:
ok will do

also, @jim-minter asked on IRC if I was good with `openshift_build_total` for the count and I said yes... will be pushing that changes as well